controllers/drplacementcontrol.go

```diff
@@ -487,6 +496,9 @@ func (d *DRPCInstance) RunRelocate() (bool, error) {
 		d.setDRPCCondition(&d.instance.Status.Conditions, rmn.ConditionAvailable, d.instance.Generation,
 			d.getConditionStatusForTypeAvailable(), string(d.instance.Status.Phase), err.Error())
 
+		rmnutil.ReportIfNotPresent(d.reconciler.eventRecorder, d.instance, corev1.EventTypeWarning,
```

Comment:
Event message can be return of `validateAndSelectCurrentPrimary` as the messages are granular from the function, the current message is generic and will not give details to the user.